bindings/nodejs/src/lib.rs

```diff
@@ -157,13 +157,13 @@ impl Operator {
     }
 
     #[napi]
-    pub async fn write(&self, path: String, content: Buffer) -> Result<()> {
+    pub async fn write(&self, path: String, content: Either<Buffer, String>) -> Result<()> {
```

Comment:
Wow, so we don't need to split them into `write_string`, nodejs will convert them?